app/assets/javascripts/components/authentication/login_form.es6.jsx

```diff
@@ -40,15 +40,15 @@ class LoginForm extends Component {
     };
     var resolve = (response) => { window.location = RouteConstants.students.index };
     var reject = (response) => { this.setState({ errors: true }) };
-    Requester.post(RouteConstants.users.login, params, resolve, reject);
+    Requester.post(ApiConstants.users.login, params, resolve, reject);
   }
 
   generateHandler(field) {
     var state = {};
-    return function(event) {
+    return (event) => {
```

Comment:
Are there places where we can't use this arrow syntax? Or is this going to replace our function(params){}.bind(this) everywhere?